model_bakery/utils.py

```diff
@@ -20,7 +20,7 @@ def import_from_str(import_string):
         return import_string
 
 
-def seq(value, increment_by=1, start=None):
+def seq(value, increment_by=1, start=None, suffix=None):
```

Comment:
Since we already have a parameter called `start`, I think would be nicer to have the new one named `end`. What do you think?

Reply:
To me, `suffix` makes more sense in the context of `str` values (which is the only valid and acceptable case for this argument). Another reason is that `end` might be confused with the "final" value of the entire sequence, even though the sequence should be "endlessly" generated.

Reply:
Ultimately, I will defer to you and @berinhard, and I am okay if you would like to change the argument to `end`.

Reply:
I agree with @timgates42 on this one. Having an `end` argument could mislead people to think that the sequence can end at some value and this won't be true. Anyway, having this `start` **and** `suffix` can also be confusing. 

Just to propose other paths: what do you think about having an explicit function to generate sequences for texts. I mean, I didn't think about this before, but maybe it would remove the need to check for `value` and `suffix` types and the caller will know that this method is a special one to handle strings. Something like:

```
seq('foo', increment_by=2, start=10, suffix='name: '): 
>>> "name: foo10
>>> "name: foo12
>>> "name: foo14
>>> "name: foo16
```

What do you think?

Reply:
But, one thing we could **definitely** improve; docstrings! They can be very helpful to people understand how to combine the parameters. @timgates42 can you add docstrings to the `seq` method?

Reply:
Yup, makes sense @berinhard. Docstrings are fine for me. 👍🏽

Reply:
Yes, I agree. I think the approach to keep one method and add docstrings makes sense. I will do that. Thanks!